flink-connectors/flink-connector-hive/src/test/java/org/apache/flink/table/functions/hive/HiveSimpleUDFTest.java

```diff
@@ -121,30 +124,30 @@ public void testUDFJson() {
 		assertEquals(null, udf.eval(json, pattern));
 	}
 
-	@Test
-	public void testUDFMinute() {
+//	@Test Disabled because in HIve 3.1.0 UDFMinute is no long a simple UDF.
```

Comment:
I'm fine just removing this test or replace it with another Hive UDF, instead of disabling it

Reply:
Ok. I will remove it.